charts/refinery/templates/deployment-marker-job.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "refinery.fullname" . }}-deploy-marker
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "refinery.labels" . | nindent 4 }}
+
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ include "refinery.serviceAccountName" . }}
+      restartPolicy: Never
+      containers:
+        - name: {{ .Chart.Name }}-deploy-marker
+          image: "{{ .Values.deployMarker.image.repository }}:{{ .Values.deployMarker.image.tag }}"
+          imagePullPolicy: {{ .Values.deployMarker.image.pullPolicy }}
+
+          {{- with .Values.environment }}
+          env:
+          {{- toYaml . | nindent 12 }}
+          {{- end }}
```

Comment:
We should add a `.Values.deployMarker.env` to use instead so this job doesn't get any extra env vars.